tests/static_multiset/custom_count_test.cu

```diff
@@ -61,21 +61,27 @@ void test_custom_count(Set& set, size_type num_keys)
 {
   using Key = typename Set::key_type;
 
+  auto const hash = []() {
+    if constexpr (cuco::is_double_hashing<typename Set::probing_scheme_type>::value) {
+      return cuco::pair{custom_hash{}, custom_hash{}};
```

Comment:
Shall we use `cuda::std::tuple` here instead?

Reply:
I’ve been thinking about this: `cuco::pair` can be constructed from tuple-like inputs, so it works with types like `cuda::std::pair/tuple` and `std::pair/tuple`. Initially, we opted for `cuco::pair` for convenience, but I’m now questioning if that’s a convenience we still need to maintain. What’s your take on this?

Reply:
I would say `cuco::pair` is only needed when interacting with table slots as it guarantees proper alignment. For any other case `cuda::std::tuple` is the more generic choice.

Reply:
> cuco::pair is only needed when interacting with table slots as it guarantees proper alignment.

Good point.

Regarding double hashing, I will go with `cuda::std::pair` since we know the number of elements must be 2.